autoload/vundle/installer.vim

```diff
@@ -527,8 +527,11 @@ func! s:log(str, ...) abort
   let fmt = '%Y-%m-%d %H:%M:%S'
   let lines = split(a:str, '\n', 1)
   let time = strftime(fmt)
+  let entry_prefix = '['. time .'] '. prefix
   for line in lines
-      call add(g:vundle#log, '['. time .'] '. prefix . line)
+    " Trim trailing whitespace
+    let entry = substitute(entry_prefix . line, '\m\s\+$', '', '')
```

Comment:
This falls apart for the case where `line` is the empty string. This would append the empty string onto the end, but wouldn't fix the trailing whitespace in `entry_prefix` (since we put a space between the time stamp and the `prefix`. So I believe this approach is a more robust way to go